pyproject.toml

```diff
@@ -13,15 +13,17 @@ requires-python = ">=3.12,<3.13"
 [project.urls]
 repository = "https://github.com/Ontotext-AD/qa-eval"
 
+[tool.poetry.dependencies]
+openai = "^1.97.0"
```

Comment:
I would mark this dependency as optional. Users wanting to use the answer evaluation script should add it as dependency to their code I think. My main objective for this suggestion is that if this is to be a widely used library, it should cause minimum collisions with users dependency trees.